test/sanity/validate-modules/validate-modules

```diff
@@ -851,6 +853,30 @@ class ModuleValidator(Validator):
 
         return False
 
+    def _check_for_subprocess(self):
+        for child in self.ast.body:
+            if isinstance(child, ast.Import):
```

Comment:
I don't think we need to look for `subprocess` imports before looking for subprocess uses.